From 3387859452604035be542a830414c6fef1d38544 Mon Sep 17 00:00:00 2001
From: "Alexander F. Lent" <lx@xanderlent.com>
Date: Tue, 14 Jan 2025 19:18:02 -0500
Subject: [PATCH 3/3] Disable insecure rpath when using system OpenVINO

Signed-off-by: Alexander F. Lent <lx@xanderlent.com>
---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index ae68d4f..9d0de75 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -125,6 +125,6 @@ add_library(intel_npu_acceleration_library SHARED src/bindings.cpp)
 
 # Link the OpenVINO libraries
 target_link_libraries(intel_npu_acceleration_library PRIVATE openvino::runtime)
-if (UNIX)
+if (UNIX AND DOWNLOAD_OPENVINO_BINARY)
     set_target_properties(intel_npu_acceleration_library PROPERTIES LINK_FLAGS "-Wl,-rpath,./")
 endif (UNIX)
-- 
2.45.2

